### PR TITLE
Docs for tica

### PR DESCRIPTION
**Pull request by clonker**

### CODE REVIEW

File: sktime/data/util.py [hunk 1]
```diff
@@ -2,6 +2,44 @@
 
 
 def timeshifted_split(inputs, lagtime: int, chunksize=1000, n_splits=None):
+    r""" Utility function which splits input trajectories into pairs of timeshifted data :math:`(X_t, X_{t+\tau})`.
+    In case multiple trajectories are provided, the timeshifted pairs are always within the same trajectory.
+
+    Parameters
+    ----------
+    inputs : (T, n) ndarray or list of (T_i, n) ndarrays
+        Input trajectory or trajectories. In case multiple trajectories are provided, they must have the dimension in
```
[the remark this answers is not in the document]
Comment (clonker) [reply sktime/data/util.py]: yes, thanks!

File: docs/source/references.bib [hunk 2]
```diff
@@ -37,3 +37,69 @@ @inproceedings{sculley2010web
...
+  author={P{\'e}rez-Hern{\'a}ndez, Guillermo and Paul, Fabian and Giorgino, Toni and De Fabritiis, Gianni and No{\'e}, Frank},
+  journal={The Journal of chemical physics},
+  volume={139},
+  number={1},
+  pages={07B604\_1},
+  year={2013},
+  publisher={American Institute of Physics}
+}
+@article{schwantes2013improvements,
+  title={Improvements in Markov state model construction reveal many non-native interactions in the folding of NTL9},
```
[the remark this answers is not in the document]
Comment (clonker) [reply docs/source/references.bib]: done

File: docs/source/references.bib [hunk 3]
```diff
@@ -37,3 +37,69 @@ @inproceedings{sculley2010web
   pages={1177--1178},
   year={2010}
 }
+@article{molgedey1994separation,
+  title={Separation of a mixture of independent signals using time delayed correlations},
+  author={Molgedey, Lutz and Schuster, Heinz Georg},
+  journal={Physical review letters},
```
[the remark this answers is not in the document]
Comment (clonker) [reply docs/source/references.bib]: right i have done it for this one but I think this is better done at the very end once

[the remark this answers is not in the document]
Comment (brookehus) [reply docs/source/references.bib]: definitely!

**Pull request merged**